python/cudf/cudf/tests/test_csv.py

```diff
@@ -851,10 +846,11 @@ def test_csv_reader_empty_dataframe():
     assert df.shape == (0, 2)
     assert all(df.dtypes == ["float64", "int64"])
 
+    # BUG - EMPTY_DATAFRAME
```

Comment:
What is the bug here?

Reply:
If the dataframe is zero row, then csv reader throws error as it couldn't find type.

Reply:
What is Pandas behavior here? Are we deviating from expected behavior now?

Reply:
no, pandas provides object type column if empty and we are doing same.